flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/KafkaOptions.java

```diff
@@ -120,6 +122,12 @@ private KafkaOptions() {}
                                     + ValueFieldsStrategy.EXCEPT_KEY
                                     + "'.");
 
+    public static final ConfigOption<String> SCHEMA_REGISTRY_SUBJECT =
```

Comment:
private scope and String type is enough. 
And please move it under all other options.